docs/dev/cli/sauce-connect-5/run.md

```diff
@@ -289,6 +305,10 @@ Setting this to direct sends requests to localhost directly without using the up
 **Environment variable**: `SAUCE_TLS_PASSTHROUGH_DOMAINS`<br/>
 **Shorthand**: `-B`
 
+```bash
+--tls-passthrough-domains .*.example.com,.*google.com,mycompany.com
```

Comment:
This would work, but `.` needs to be escaped or it's a wildcard, eg. `.*\.example\.com`